test/e2e/data/nodefeature-3.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeature
+metadata:
+  name: zzz-e2e-features-3
+spec:
+  labels:
+    random.denied.ns/e2e-nodefeature-test-1: "denied-ns"
+    random.unwanted.ns/e2e-nodefeature-test-2: "unwanted-ns"
+    custom.vendor.io/e2e-nodefeature-test-3: "vendor-ns"
+    e2e-nodefeature-test-4: "obj-4"
```

Comment:
You could add `kubernetes.io/denied-label` and `subns.kubernetes.io/denied-label` here to verify that these really are denied by default